### PR TITLE
[BALDE-16] Add Vector3 (x, y, z)

### PR DESCRIPTION
**Pull request by 4jeR**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -72,7 +72,7 @@ cotire(bald_engine)
 enable_testing()
 file(GLOB TEST_SRC_FILES ${PROJECT_SOURCE_DIR}/tests/*.cpp src/Log.cpp)
 
-add_executable(runUnitTests ${TEST_SRC_FILES})
+add_executable(runUnitTests ${TEST_SRC_FILES} tests/test_vector3.cpp)
```
Comment (piotrmoszkowicz) [inline CMakeLists.txt]: Not needed here, I know CLion adds it by itself, but remember about it please 😄

File: tests/test_vector3.cpp [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+
+TEST(NormalizeVector, Vec3_StaticUnitVec) { //NOLINT
+    Bald::Math::Vec3 A = Bald::Math::Vec3::MakeUnitVec(Bald::Math::Vec3(2.0f, 3.0f, 6.0f));
+
+    float x = 2.0f / 7.0f; // 0.28571
+    float y = 3.0f / 7.0f; // 0.42857
+    float z = 6.0f / 7.0f; // 0.85714
+
+    ASSERT_EQ(Bald::Math::Vec3(x, y, z), A);
```
Comment (piotrmoszkowicz) [inline tests/test_vector3.cpp]: Why not move equations into the Vec3 constructor?

**Review by piotrmoszkowicz — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (4jeR) [reply CMakeLists.txt]: fixed

File: tests/test_vector3.cpp (hunk 2 above)
Comment (4jeR) [reply tests/test_vector3.cpp]: moved

**Pull request closed without merging**